asset_tracker_app/lib/view/login_screen_view.dart

```diff
@@ -0,0 +1,190 @@
+import 'package:asset_tracker_app/utils/constants.dart';
+import 'package:flutter/material.dart';
+
+class LoginScreenView extends StatefulWidget {
+  const LoginScreenView({super.key});
+
+  @override
+  State<LoginScreenView> createState() => _LoginScreenViewState();
+}
+
+class _LoginScreenViewState extends State<LoginScreenView> {
+  final _formKey = GlobalKey<FormState>();
+  final _emailController = TextEditingController();
+  final _passwordController = TextEditingController();
+  final _authService = AuthService();
+  bool _isPasswordVisible = false;
+  bool _isLoading = false;
+  bool _isLogin = true; // true = login, false = signup
+
+  @override
+  void dispose() {
+    _emailController.dispose();
+    _passwordController.dispose();
+    super.dispose();
+  }
+
+  Future<void> _submitForm() async {
+    if (_formKey.currentState?.validate() ?? false) {
+      setState(() => _isLoading = true);
+      try {
+        if (_isLogin) {
+          final success = await _authService.signIn(
+            _emailController.text,
+            _passwordController.text,
+          );
+
+          if (mounted && success) {
+            Navigator.pushReplacementNamed(context, ToScreen.homePage);
+          }
+        } else {
+          final success = await _authService.signUp(
+            _emailController.text,
+            _passwordController.text,
+          );
+
+          if (mounted && success) {
+            ScaffoldMessenger.of(context).showSnackBar(
+              const SnackBar(
+                content: Text('You have successfully registered.'),
+                backgroundColor: Colors.green,
+                duration: Duration(seconds: 2),
+              ),
+            );
+
+            await Future.delayed(const Duration(seconds: 2));
+            if (mounted) {
+              setState(() {
+                _isLogin = true;
+                _emailController.clear();
+                _passwordController.clear();
+              });
+            }
+          }
+        }
+      } catch (e) {
+        if (mounted) {
+          ScaffoldMessenger.of(context).showSnackBar(
+            SnackBar(
+              content: Text(e.toString()),
+              backgroundColor: Colors.red,
+              duration: const Duration(seconds: 3),
+            ),
+          );
+        }
+      } finally {
+        if (mounted) {
+          setState(() => _isLoading = false);
+        }
+      }
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SafeArea(
+        child: SingleChildScrollView(
+          padding: const EdgeInsets.all(24.0),
+          child: Form(
+            key: _formKey,
+            child: Column(
+              crossAxisAlignment: CrossAxisAlignment.stretch,
+              children: [
+                const SizedBox(height: 48),
+                Icon(
+                  Icons.inventory,
+                  size: 80,
```

Comment:
yükseklik veya genişlik için kullanacağın boşlukları tek bir sınıfta yönetebilirsin. Bu sayede değişime açık bir kodun olur. 24 yazmak yerine  DeviceSize.big gibi gibi